src/components/Channel/Channel.tsx

```diff
@@ -687,6 +687,13 @@ const ChannelInner = <
     event: React.BaseSyntheticEvent,
   ) => {
     event.preventDefault();
+    // This is the desired behavior I want to achieve
```

Comment:
I don't understand why quoted message is undefined here, despite having a message selected for quote reply, can you also help with this?

Reply:
Hmm, I see this - I did some poking around, and I'm not sure either. Maybe @petyosi can assist?